action.yml

```diff
@@ -567,13 +571,28 @@ runs:
         echo "No builder was able to build the provided application source. Please visit the following page for more information on supported platform versions: https://aka.ms/SourceToCloudSupportedVersions"
         exit 1
 
+    - name: Convert build arguments to string
+      if: ${{ inputs.buildArguments != '' }}
+      shell: bash
+      run: |
+        build_arguments=""
+
+        IFS=' ' read -ra build_arguments_array <<< "${{ inputs.buildArguments }}"
+
+        for build_argument in "${build_arguments_array[@]}"; do
+          build_arguments+=" --build-arg $build_argument"
+        done
+
+        echo "CA_GH_ACTION_BUILD_ARG=${build_arguments}" >> $GITHUB_ENV
```

Comment:
Let's update the environment variable we're setting here to `CA_GH_ACTION_DOCKERFILE_BUILD_ARG` just to be crystal clear what the build arguments are for.